tests/NSubstitute.Analyzers.Tests.Shared/NSubstitute.Analyzers.Tests.Shared.csproj

```diff
@@ -7,6 +7,7 @@
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="coverlet.msbuild" Version="2.6.0" />
```

Comment:
using coverlet instead of OpenCover for calculating code coverage. OpenCover doesn't support Linux https://github.com/OpenCover/opencover/issues/703